src/main/java/org/irods/Util/Acl.java

```diff
@@ -0,0 +1,20 @@
+package org.irods.Util;
+
+/**
+ * enum for modify_permission() method to ensure user only inputs valid data
+ * */
+public enum Acl {
```

Comment:
ACL (access control list) and Permissions serve the same purpose.

Let's remove `Acl` and use the `Permission` enumeration.

Then the modify_permissions operation can simply throw an exception if the user passes an enumeration that isn't supported.

Reply:
Done. I made an issue to add a test to make sure it throws an exception. 
Created #80